test/testrunner.py

```diff
@@ -123,18 +124,46 @@ def main():
                     f"Instead seeing {frameworks_in_images} frameworks."
                 )
             framework = frameworks_in_images[0]
+
+            # Separate multi-node EKS tests from single-node tests in execution to prevent resource contention
+            multi_node_tests_per_framework = {
```

Comment:
We shouldn't hardcode the test names in a separate location -- if the name changes for some reason or a new multinode test is added, we'll have to make a change in two places, making it easy for issues to arise. We should use pytest for things like this.

Take a look at how the canary tests are run. We already have a pytest marker called "multinode" -- you can edit pytest_runtest_setup and add a new option called "--multinode" and mark the multinode eks tests as multinode (they will be remarked in the pytest marker PR but it's fine to add some to a few tests for now); then in your pytest command, you would just call `pytest -s -rA eks/ --multinode --junitxml..`

Reply:
Ah, good call out. I'll make this change.